<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>685</height>
    </rect>
   </property>
   <widget class="QPushButton" name="sample_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>320</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string>Sample Generation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filter_acts_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>320</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter Activations</string>
    </property>
   </widget>
   <widget class="QPushButton" name="layer_rep_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>360</y>
      <width>320</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer Representatiom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grouped_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>480</y>
      <width>320</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string>Grouped Layer Representation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_model_btn">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>250</y>
      <width>190</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_input_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>370</y>
      <width>190</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Input</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>955</x>
      <y>200</y>
      <width>190</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Dictionary</string>
    </property>
   </widget>
   <widget class="QPushButton" name="import_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>955</x>
      <y>310</y>
      <width>190</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Dictionary</string>
    </property>
   </widget>
   <widget class="QLabel" name="loading_lbl">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>620</y>
      <width>65</width>
      <height>50</height>
     </rect>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Loading...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="import_settings_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>955</x>
      <y>420</y>
      <width>190</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
